base_transaction_id/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+============================================
+Base transaction id for financial institutes
+============================================
+
+Adds transaction id to invoice and sale models and views.
+
+On Sales order, you can specify the transaction ID used
+for the payment and it will be propagated to the invoice (even if made from packing).
+This is mostly used for e-commerce handling.
+
+You can then add a mapping on that SO field to save the e-commerce financial
+Transaction ID into the Odoo sale order field.
+
+The main purpose is to ease the reconciliation process and be able to find the partner
+when importing the bank statement.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/bank-statement-reconcile/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
+* Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Pedro Baeza <pedro.baezo@gmail.com>
```

Comment:
is that email correct?

Reply:
I removed it as he didn't contributed directly to that module, he has few commits through OCA module maintenance

Reply:
Yeah, I haven't made anything significant here.